src/pages/learn.js

```diff
@@ -275,10 +275,15 @@ const LearnPage = ({
         [filterValue]
     );
 
-    const activeContentTab = useMemo(
-        () => filterValue['content'] || LearnPageTabs.all,
-        [filterValue]
-    );
+    const activeContentTab = useMemo(() => {
+        const currentContentFilter = filterValue['content'];
+        if (currentContentFilter) {
+            return (
+                LearnPageTabs[filterValue['content']] || filterValue['content']
+            );
```

Comment:
the nav uses ?content=articles, which does not exact match the tab value of `Articles` (uppercase). We use an enum for these and since we could also match the key (lowercase) this fixes it. Makes sense?

Reply:
Did we give this as landing page to the nav team ? https://www.mongodb.com/developer/learn/?content=articles#main, why din't we just give https://www.mongodb.com/developer/

Reply:
This was the design goal, for us to serve the learn page directly since that is where they can access most of the content.

It is a bit odd I agree, but with the new navigation the home page is not as useful so they want to serve the users this page directly. That is expected.

Reply:
In this case isn't the main URL : https://www.mongodb.com/developer/learn/

Reply:
Yes, but the `#main` brings the user down the page